### ISSUE TITLE
single protein-phosphorylaation

### ISSUE THREAD

**Issue by haiyan-MS** [vdemichev/DiaNN#640]
Hi 

I am just starting to use DIA NN. It has been producing great results for us when we did cell lysate. 
I recently did a project with in vitro phosphorylation. I have 4 samples. A and B were mostly a human protein with a human kinease (minute amount) but some residual e.coli proteins. C and D are mostly E.coli protein with very few human protein and kinase (minute amount). I did DIA and searched using DIA NN 1.8.1. I used library free search with E.coli database and a small database with two sequences (human substrate protein and human kinase sequences) with dismal results. In DDA and base peaks, I can see this human protein clearly and covarage should be >90%. But in DIA NN, A and B produced about 10% coverage of the human substrate. Manual inspection of the raw files looks fine. I would like to find out where I did wrong. 
![image](https://user-images.githubusercontent.com/91212783/226366099-a2690330-b9e5-4f41-af49-3de255b8c3dc.png)

**Comment by vdemichev** [comment]
The settings of DIA-NN do not follow the guildeline https://github.com/vdemichev/DiaNN#changing-default-settings, however this would not affect the results significantly in this case. Can you please share the full log and screenshots demonstrating what you would expect to see but don't see in DIA-NN output?

**Comment by haiyan-MS** [comment]
Dear Professor Demichev:

Thank you so much for this wonderful software. We had tried many proteomics projects. DIA NN always produce good results. But for this in vitro experiment, it was disappointing.
I am concerned about the 1st comment you have. I did not change anything except checked phosphorylation. Why do you say it did not follow guideline.
I attached pipeline (not sure where the pepline was saved so I copy pasted into notepad) and the human protein sequences I used for searching.
I am also sending screenshot of  basepeak comparison between DIA and DDA for one of the samples and the MSMS of several peptides that were not identified in DIA and compared to DDA (all identified).
In DDA, the human substrate OPA1 is identified and coverage is >90%. But in DIA I would say only about 15 to 20%.

Let me know what else you need to find out why I can't identify any phosphorylation and why the coverage for OPA1 is so low in DIA run.

To reiterate, this is invitro experiment of a human substrate overexpressed in Ecoli and purifed  plus commercial human kinase to study in vitro phosphorylation.I searched e.coli fasta database plus a text file containing the two human protein sequences that were downloaded from Uniprot.

Thank you again for taking the time to look into this.

Haiyan


[cid:51d34b5a-f628-4934-9104-988997aca1e0]
[cid:61b56347-d290-41dd-9767-ad226129fd53]
[cid:6a68a1f4-a9ba-4d80-afb9-81cbea25cfae]
________________________________

**Comment by vdemichev** [comment]
Hi Haiyan,

Ox(M) and double-pass mode are not recommended.
For most phospho workflows, setting the number of variable modifications to 3 is sensible.

The log attached is not consistent with the screenshot on the first post. That is those were obtained with different settings. Specifically, the screenshot shows analysis log obtained without 'Phospho' selected. So when you refer to DIA-NN output, which way it was obtained? I would suggest to reanalyse everything without reusing .quant files, to make sure all the settings are good.

Best,
Vadim

**Comment by vdemichev** [comment]
Can you please attach on github? Not so convenient via email.
How were you getting ~10k IDs on a screenshot in the initial post, i.e. why more IDs then?
Based on the log, looks like only few if any peptides are phosphorylated. Does the first-pass report contain the peptides of interest?

**Comment by haiyan-MS** [comment]
[report-DIA_MS6018.log.txt](https://github.com/vdemichev/DiaNN/files/11041976/report-DIA_MS6018.log.txt)
[report-DIA_MS6018.pdf](https://github.com/vdemichev/DiaNN/files/11041979/report-DIA_MS6018.pdf)

Please let me know if this is better. 

There might be a little more ID, but not much more. I guess single pass instead of double? No M oxi? I only searched two runs instead of 4 runs?

**Comment by vdemichev** [comment]
"There might be a little more ID, but not much more" - but >9k here? 

![image](https://user-images.githubusercontent.com/30595084/226966164-314dd5ba-2ea1-40df-ae10-12638db0a9fd.png)

**Comment by haiyan-MS** [comment]
I just looked the 1st pass report. THe coverage is 89% and there are tons of phosphorylation! I will have a closer look of this file. Do you suggest I use this one instead of the final report?

**Comment by haiyan-MS** [comment]
I have tried this search with generated library. But somehow, I don't know why it only take one lib but not two. 
Also, I have a question, when using this library, do I deselect all the modifications in the default? Do I still select phosphorylation or not? Please have a look of my search log. And also my libraray I generated with the two human proteins. 
[report-DIA_libsearch.log.txt](https://github.com/vdemichev/DiaNN/files/11044338/report-DIA_libsearch.log.txt)
![image](https://user-images.githubusercontent.com/91212783/227024929-e3621ebe-58ec-4eca-a6ed-d545517743c3.png)

**Comment by vdemichev** [comment]
Predicted libraries need to be converted to .tsv first.

**Comment by vdemichev** [comment]
Please see above: 'Check the 'Phospho' checkbox.'

**Comment by haiyan-MS** [comment]
[report-lib-4.log.txt](https://github.com/vdemichev/DiaNN/files/11054458/report-lib-4.log.txt)
This time there is 0 results

**Comment by vdemichev** [comment]
Yes, need to convert .predicted.speclib libraries into .tsv, please see above.

**Comment by haiyan-MS** [comment]
[OPA1.log.txt](https://github.com/vdemichev/DiaNN/files/11054614/OPA1.log.txt)
I don't see .tsv although I said generate OPA1.tsv 
![image](https://user-images.githubusercontent.com/91212783/227311908-835ff718-36e2-4701-95d7-a9105577774f.png)

**Comment by vdemichev** [comment]
Yes, now need to 1. Reset all settings. 2. put OPA1.predicted.speclib into the 'Spectral library' field. 3. Check 'Generate spectral library'.

**Comment by haiyan-MS** [comment]
Still the samething. only the 1st lib was used. 
[report-again.log.txt](https://github.com/vdemichev/DiaNN/files/11062374/report-again.log.txt)
I accidentally loaded all 4 runs. This time, only the last two files give results. 
But the basic issue is it only used 1st lib.

**Comment by haiyan-MS** [comment]
But this time I converted to tsv

**Comment by haiyan-MS** [comment]
Found the mistake. I still have the E.coli lib selected in the window where you can put in lib file if you want single lib. I will try again.

**Comment by haiyan-MS** [comment]
Professor Demichev:

Thank you for your help. It finally worked and give me a lot of overage. This issue is resolved. Of course I need to have a look more carefully about them and to see if they make sense. I might come back to ask you.

**Comment by haiyan-MS** [comment]
I have one question: Is there a way to get observed MS value in the report? Or even therotical value?

**Comment by vdemichev** [comment]
Theoretical - can just match the preursors to the library entries. Empirical - currently not supported.

**Comment by haiyan-MS** [comment]
Yes. But how do I get it out? Of course I can caculate individually, but it will be best to have it in the report.
